english/quick_draw/AI_Pictionary.md

```diff
@@ -1,3 +1,5 @@
 It is a lot of fun playing pictionary with a bunch of friends. 
+
 But have you ever played pictionary with a computer, where you draw sketches and computer predicts your sketch?
-If not, play with this awesome [AI](https://quickdraw.withgoogle.com/) built by guys at Google.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.